msal-dotnet-articles/acquiring-tokens/desktop-mobile/linux-broker-net.md

```diff
@@ -0,0 +1,149 @@
+---
+# Required metadata
+# For more information, see https://review.learn.microsoft.com/en-us/help/platform/learn-editor-add-metadata?branch=main
+# For valid values of ms.service, ms.prod, and ms.topic, see https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main
+
+title:       Use MSAL.NET with an authentication broker on Linux
+description: "Learn how to use MSAL.NET with a broker on the Linux platform."
+author:      xinyuxu1026 # GitHub alias
+ms.author:   xinyuxu # Microsoft alias
+ms.service:  msal
+# ms.prod:   # To use ms.prod, uncomment it and delete ms.service
+ms.topic:    how-to
+ms.date:     05/05/2025
+---
+
+# Use MSAL.NET with an authentication broker on Linux
+
+
+> [!NOTE]
+> Microsoft single sign-on (SSO) for Linux authentication broker support is introduced with `Microsoft.Identity.Client` version v4.69.1.
+
+Using an authentication broker on Linux enables you to simplify how your users authenticate with Microsoft Entra ID from your application, as well as take advantage of future functionality that protects Microsoft Entra ID refresh tokens from exfiltration and misuse.
+
+An authentication broker is **not** pre-installed on standalone Linux but is bundled as a dependency of applications developed by Microsoft, such as [Company Portal](/mem/intune-service/user-help/enroll-device-linux). These applications are usually installed when a Linux computer is enrolled in a company's device fleet via an endpoint management solution like [Microsoft Intune](/mem/intune/fundamentals/what-is-intune). For [Windows Subsystem for Linux](/windows/wsl/about) (WSL) scenario, WAM (Windows Account Manager) is used as the broker. WAM does come pre-installed on the Windows system. To learn more about Linux device set up with the Microsoft Identity Platform, see [Microsoft Enterprise SSO plug-in for Apple devices](/entra/identity-platform/apple-sso-plugin).
+
+## Prerequisites
+
+MSAL.NET project requires .NET version greater or equal than the version specified in [global.json](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/global.json) file, we recommend register the Microsoft package repository and install .NET, please follow the instructions here: [Install .NET on Ubuntu](/dotnet/core/install/linux-ubuntu-decision#register-the-microsoft-package-repository). And please make sure you have all the [.NET required dependencies](/dotnet/core/install/linux-ubuntu-decision#dependencies) installed.
```

Comment:
It isn't clear to me why I should care about what the MSAL.NET project requires.

Reply:
I like the part in which it recommends registering the Microsoft Package Repo and install .NET from it, cause Ubuntu has its own feed and it took me a while to install net8.0.4.